BritneyS/StarWarsDetail
# Add detail ui

## What you did :question:
- Populated the Character Detail View with the data returned in segue and API call

## How you did it :white_check_mark:
- Added methods to populate the label text of all property labels
- For species label: returning more than one species string results in multiple species listed


## How to test it :microscope:
- Run the app
- Tap any character name
- Character Detail View should appear, showing the name, birth year, gender, homeworld, and species of the selected character


## Screenshots (if applicable) :camera:
![simulator screen shot - iphone xr - 2018-10-27 at 20 02 36](https://user-images.githubusercontent.com/8409475/47610493-9d726d80-da24-11e8-8ef4-460a80db0c40.png)
![simulator screen shot - iphone xr - 2018-10-27 at 20 02 47](https://user-images.githubusercontent.com/8409475/47610494-a6fbd580-da24-11e8-92d7-6653c97d1eb2.png)
